src/main/java/com/github/krzysiekjodlowski/ox/Symbol.java

```diff
@@ -0,0 +1,30 @@
+package com.github.krzysiekjodlowski.ox;
+
+/**
+ * Represents each player on Board.
+ * @author Krzysztof Jodlowski
+ */
+enum Symbol {
+    NAUGHT("O"), CROSS("X");
+
+    /**
+     * Holds String representation of an enum.
+     */
+    private final String value;
+
+    /**
+     * Constructor forced by language.
+     * @param value initializes enum value
+     */
+    Symbol(final String value) {
+        this.value = value;
+    }
+
+    /**
+     * Useful in board representation.
+     * @return String representation of an enum
+     */
+    String getValue() {
```

Comment:
Maybe use toString instead? Because for now you will have X printed when you use getValue, but if you forget to add it, you will get CROSS printed instead. So, unless it is an expected behaviour, I would be carefull with current design.

Reply:
Good idea! Done.